brands/shop/hardware.json

```diff
@@ -130,4 +130,17 @@
       "shop": "hardware"
     }
   }
-}
\ No newline at end of file
+  "shop/hardware|Home Hardware": {
+    "countryCodes": ["my"],
+    "nomatch": [
+      "shop/doityourself|MR.DIY"
```

Comment:
Is "shop/doityourself|MR.DIY" added already to NSI if it is also a valid value?

Reply:
And if `shop=doityourself` should be replaced by `shop=hardware` then nomatch should be replaced by match